R/pkg/NAMESPACE

```diff
@@ -10,6 +10,10 @@ export("sparkR.init")
 export("sparkR.stop")
 export("print.jobj")
 
+# MLlib integration
+export("glm")
```

Comment:
Just realized that we also change to use method for them, then this should be `exportMethods()`